gettsim/synthetic_data/synthetic.py

```diff
@@ -185,8 +185,7 @@ def create_one_set_of_households(
     policy_year,
     **kwargs,
 ):
-    """ creates one set of households
-    """
+    """creates one set of households"""
```

Comment:
```suggestion
    """Create one set of households."""
```

Reply:
Maybe add a sentence? Unrelated, sorry, but it is unclear without context what exactly is supposed to happen here.